src/node/hooks/express/webaccess.js

```diff
@@ -11,6 +11,15 @@ const cookieParser = require('cookie-parser');
 
 hooks.deprecationNotices.authFailure = 'use the authnFailure and authzFailure hooks instead';
 
+const staticPathsRE = new RegExp('^/(' + [
```

Comment:
I'm not sure that will be any more readable, especially if we want to be conservative on what we permit (e.g., exempt `/static/foo` and `/favicon.ico` from auth checks but not `/static` or `/favicon.ico/foo`). It also wouldn't accommodate paths like `/topdir/subdir/*` if we ever want to add something like that to this list.

Reply:
Okay, well my rule is just to be expressive w/ RE's.  I'd say this is expressive enough but please just try to avoid REs in teh future as I find the technical debt of them is often much higher than writing a bunch of cases / ifs etc.